src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/auth/AuthApiService.kt

```diff
@@ -120,3 +124,35 @@ data class CreateTokenRequest(
   val firstName: String,
   val lastName: String,
 )
+
+data class AuthUserDto(
+  @Schema(description = "User ID", example = "91229A16-B5F4-4784-942E-A484A97AC865")
+  val userId: String? = null,
+
+  @Schema(description = "Username", example = "externaluser")
+  val username: String? = null,
```

Comment:
Dto now mirrors Auth with nullable fields and corrected addition of "source" which did not appear in original Dto and removal of inactiveReason which should not appear